meta-balena-zeus/recipes-core/util-linux/util-linux/lsblk-force-to-print-PKNAME-for-partition.patch

```diff
@@ -0,0 +1,32 @@
+From e3bb9bfb76c17b1d05814436ced62c05c4011f48 Mon Sep 17 00:00:00 2001
+From: Karel Zak <kzak@redhat.com>
+Date: Thu, 27 Jun 2019 09:22:18 +0200
+Subject: [PATCH] lsblk: force to print PKNAME for partition
+
+PKNAME (parent kernel device name) is based on printed tree according
+to parent -> child relationship. The tree is optional and not printed
+if partition specified (.e.g "lsblk -o+PKNAME /dev/sda1"), but old
+versions print the PKNAME also in this case.
+
+Upstream-Status: Backported
```

Comment:
This also needs a `Signed-off-by` yourself and moved just before the patch begins. See https://github.com/balena-os/balena-raspberrypi/blob/master/layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi/0001-Fbcon-ignore-events-for-rpi-sense-fb.patch#L18-L19 for example